internal/controller/postgrescluster/pod_disruption_budget_test.go

```diff
@@ -2,18 +2,18 @@
 // +build envtest
 
 /*
- Copyright 2021 - 2022 Crunchy Data Solutions, Inc.
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
+Copyright 2021 - 2022 Crunchy Data Solutions, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
 
- http://www.apache.org/licenses/LICENSE-2.0
+http://www.apache.org/licenses/LICENSE-2.0
 
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
 */
 package postgrescluster
```

Comment:
🔧 Our [hack/boilerplate.go.txt](https://github.com/CrunchyData/postgres-operator/blob/master/hack/boilerplate.go.txt) has the leading space. I think `go fmt` is changing this because it's incorrectly on the package. There should be a blank line above the package clause.

```suggestion
 limitations under the License.
*/

package postgrescluster
```

Reply:
:+1: That seems to have fixed it, updated.